.storybook/preview.tsx

```diff
@@ -30,9 +31,11 @@ const preview: Preview = {
     Story => (
       <SiteProvider>
         <LocaleProvider>
-          <div className={rootClasses}>
-            <Story />
-          </div>
+          <NotificationProvider viewPortClassName="absolute top-0 left-0 list-none">
```

Comment:
```suggestion
          <NotificationProvider viewportClassName="absolute top-0 left-0 list-none">
```

viewport is one word afaik

Reply:
updated